pkg/controller/apimanager/upgrade_26_to_27.go

```diff
@@ -379,3 +385,29 @@ func (u *Upgrade26_to_27) upgradeImageStreams() (reconcile.Result, error) {
 
 	return reconcile.Result{Requeue: anImageStreamChanged}, nil
 }
+
+func (u *Upgrade26_to_27) upgradeSystemRollingUpdatesCfgMap() (reconcile.Result, error) {
+	cfgMapName := "system"
+	cfgMapEntry := "rolling_updates.yml"
+	expectedValue := "api_as_product: true"
+
+	systemCfgMap := &v1.ConfigMap{}
+	err := u.client.Get(context.TODO(), types.NamespacedName{Name: cfgMapName, Namespace: u.cr.Namespace}, systemCfgMap)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+
+	var rollingUpdatesTxt string
+	rollingUpdatesTxt, ok := systemCfgMap.Data[cfgMapEntry]
+	if !ok {
+		return reconcile.Result{}, fmt.Errorf("Expected %s key in %s ConfigMap not found", cfgMapEntry, cfgMapName)
+	}
+
+	if !strings.Contains(rollingUpdatesTxt, expectedValue) {
+		u.logger.Info("%s ConfigMap %s entry updated with '%s' appended", cfgMapName, cfgMapEntry, expectedValue)
```

Comment:
structured logging [logr](https://github.com/go-logr/logr) does not work like that. It compiles, but when you run it, you get `odd number of arguments passed as key-value pairs for logging`

Reply:
Totally true.
I was thinking on fmt.Printf when running it. I'll add fmt.Printf then inside the Info if that's ok